pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -687,3 +670,24 @@ func originalAppFrom(ctx context.Context) (*v1beta1.Application, bool) {
 	app, ok := ctx.Value(OriginalAppKey).(*v1beta1.Application)
 	return app, ok
 }
+
+func newGCOptions(gcOutdated bool, revisionLimit int) []resourcekeeper.GCOption {
```

Comment:
both `isWorkflowUpdate` and  `gcOutdated` are bool. there name should have same style, since you want to. enhanced code readability.

`isWorkflowUpdate` and  `isGcOutdate`  /  `workflowUpdated` and  `gcOutdated`